WebApiBackend/WebApiBackendTests/ChoreControllerTest.cs

```diff
@@ -56,17 +58,82 @@ public void Setup()
         }
 
         /// <summary>
-        /// Ensures that a payment can be created for a flat
+        /// Checks for the equality of Chores. This is here because the functionality for checking value equality
+        /// of Chores is unavailable. Because of the rules for Pull Requests, it cannot be added in the same PR as
+        /// the tests and must be added to the Chore class later, if required elsewhere.
+        /// </summary>
+        private bool ChoreEquals(Chore choreOne, Chore choreTwo)
+        {
+            return (choreOne.Id.Equals(choreTwo.Id) &&
+               choreOne.Title.Equals(choreTwo.Title) &&
+               choreOne.Description.Equals(choreTwo.Description) &&
+               choreOne.AssignedUser.Id.Equals(choreTwo.AssignedUser.Id) &&
+               choreOne.DueDate.Equals(choreTwo.DueDate) &&
+               choreOne.Completed.Equals(choreTwo.Completed) &&
+               choreOne.Recurring.Equals(choreTwo.Recurring)
+               );
+        }
+
+        /// <summary>
+        /// Ensures that a chore can be created for a flat
         /// </summary>
         [Test]
         public async Task TestCreateChoreForFlatAsync()
+        {
+            Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
+            Trace.AutoFlush = true;
+            
+            // Arrange
+            var chore = new ChoreDTO
+            {
+                Title = "lawn",
+                Description = "mow the lawn",
+                Assignee = 1,
+                DueDate = new DateTime(2020, 05, 05),
+                Completed = false,
+                Recurring = true,
+            };
+
+            int userId = 1; // as initialised in HttpContextHelper
```

Comment:
See GitHub actions warning from build `The variable 'userId' is assigned but its value is never used`